pkg/vault/contracts/test/ProtocolFeeControllerMock.sol

```diff
@@ -18,4 +18,8 @@ contract ProtocolFeeControllerMock is ProtocolFeeController {
     ) external view returns (address poolCreator, uint256 creatorSwapFeePercentage, uint256 creatorYieldFeePercentage) {
         return (_poolCreators[pool], _poolCreatorSwapFeePercentages[pool], _poolCreatorYieldFeePercentages[pool]);
     }
+
+    function manualSetPoolCreator(address pool, address poolCreator) external {
```

Comment:
Would add a comment here that Standard Balancer Pools specifically disallow pool creators to be passed in through PoolRoleAccounts; otherwise, this wouldn't be necessary.